symfony/framework-bundle/3.3/manifest.json

```diff
@@ -14,7 +14,9 @@
     "env": {
         "APP_ENV": "dev",
         "APP_DEBUG": "1",
-        "APP_SECRET": "%generate(secret)%"
+        "APP_SECRET": "%generate(secret)%",
+        "#TRUSTED_PROXIES": "0.0.0.0/0",
```

Comment:
Maybe can you add:

    "#", "See https://symfony.com/doc/current/request/load_balancer_reverse_proxy.html",